src/main/java/com/stripe/net/RequestOptions.java

```diff
@@ -1,26 +1,35 @@
 package com.stripe.net;
 
 import com.stripe.Stripe;
+import java.util.Map;
 
 public class RequestOptions {
   public static RequestOptions getDefault() {
-    return new RequestOptions(Stripe.apiKey, Stripe.clientId, Stripe.apiVersion, null, null,
+    return new RequestOptions(Stripe.apiKey, Stripe.clientId, null, null, null,
         Stripe.getConnectTimeout(), Stripe.getReadTimeout());
   }
 
   private final String apiKey;
   private final String clientId;
-  private final String stripeVersion;
+  /**
+   * Stripe version always set at {@link Stripe#API_VERSION}.
+   */
+  private final String stripeVersion = Stripe.API_VERSION;
+  /**
+   * Stripe version when made on behalf of others. This can be used when the returned response
+   * will not be deserialized into the current classes pinned to {@link Stripe#VERSION}.
+   */
+  private final String stripeVersionOnBehalfOf;
```

Comment:
Generalize the stripe version of mobile client required by ephemeral key to `stripeVersionOnBehalfOf`.
Putting in request option here avoid considering stripe version here has method params.